products/sle/main.pm

```diff
@@ -1240,6 +1240,16 @@ elsif (get_var("XFSTESTS")) {
     }
 }
 elsif (get_var("VIRT_AUTOTEST")) {
+    if (get_var('REPO_0_TO_INSTALL', '')) {
+        #Before host installation starts, swtich to version REPO_0_TO_INSTALL if it is set
+        #Save VERSION TO TARGET_DEVELOPING_VERSION
+        set_var('TARGET_DEVELOPING_VERSION', get_var('VERSION'));
```

Comment:
Ok, I understand the name but maybe it would be more helpful to be explicit about which part of the test system the version applies to. To my understanding "TARGET_DEVELOPING_VERSION" could be "VERSION_HYPERVISOR_HOST". Is it correct? Does it sound better? Just an opinion :)

Reply:
Well, this parameter is to store the VERSION passed down from trigger script, and the meaning of it is the current developing version of product that is now being tested. It is not "VERSION_HYPERVISOR_HOST". What's more the hypervisor host version changes after host upgrade if it has such test step.

Reply:
Sure, I know where the value of VERSION comes from, from the "isos post" trigger. But to what does it relate in the test?

What I don't understand in your verification run in http://10.67.18.220/tests/422#step/welcome/1 is that VERSION=15 so "TARGET_DEVELOPING_VERSION" would also be 15 but you install a SLE12-SP3 with a kvm hypervisor and if I understand correctly you create a guest which is in this case also SLE12-SP3. So which part of the system actually runs SLE15? Maybe I just don't see it in the test run and we would just see it later when the guest *or* the host should be upgraded to SLE15? In this case maybe the name "VERSION_HYPERVISOR_HOST_MIGRATION_TARGET"? It's a bit long but should be unambiguous at least :)

Reply:
Yeah, you are right. From the verification job, i didn't use version 15. But actually in our sle15 testing, there is no such test , which install sle12sp3 on sle12sp3 host. What we have that  map to it is actually install sle15 guest on sle12sp3 host, so does sle11sp4 host. Why I use it as verification job is that sle15 guest installation via autoyast has a bug, https://bugzilla.suse.com/show_bug.cgi?id=1063939, so to let you see code path runs well and final result as pass, i give that as verification.